changelog/7307.trivial.rst

```diff
@@ -0,0 +1,3 @@
+Progress from :meth:`sunpy.net.jsoc.JSOCClient.fetch` is now reported through the logger.
```

Comment:
Does this mean no progress bar any more? Does the progress show by default? How do I turn on and off the progress now compared to before? (are all questions that I would probably ask as a user looking at this change)

Reply:
Resolving this one as the same comment seems to have been posted twice.